include/IECoreScene/Primitive.h

```diff
@@ -98,7 +99,7 @@ class IECORESCENE_API Primitive : public VisibleRenderable
 		/// transparent access to the indexed data without actually copying and expanding.
 		template<typename T>
 		typename T::Ptr expandedVariableData( const std::string &name, PrimitiveVariable::Interpolation requiredInterpolation=PrimitiveVariable::Invalid,
-			bool throwIfInvalid = false) const;
+			bool throwIfInvalid = false, const IECore::Canceller *canceller = nullptr ) const;
```

Comment:
I hadn't really anticipated cancellers getting propagated this deep into the API. Do you think this is a sign that we should be using an alternate, more implicit approach like `Context::current()` in Gaffer? Not that I think we should get into such a redesign in this PR...

Reply:
I've gotten rid of this now that we're using an IndexedView in FaceArea.  I did add a Canceller to loadPrimitiveVariables though, since it is a substitute for Object::load(), which you identified as somewhere you did want a Canceller.